presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java

```diff
@@ -122,7 +122,7 @@
     @Option(name = "--execute", title = "execute", description = "Execute specified statements and exit")
     public String execute;
 
-    @Option(name = "--output-format", title = "output-format", description = "Output format for batch mode [ALIGNED, VERTICAL, CSV, TSV, CSV_HEADER, TSV_HEADER, NULL] (default: CSV)")
+    @Option(name = "--output-format", title = "output-format", description = "Output format for batch mode [ALIGNED, VERTICAL, NO_HEADER_WITH_QUOTES, TSV, HEADER_WITH_QUOTES, TSV_HEADER, NO_HEADER_WITHOUT_QUOTES, HEADER_WITHOUT_QUOTES, NULL] (default: NO_HEADER_WITH_QUOTES)")
```

Comment:
Please do not rename options that existed before for backward-compatibility

Reply:
Sorry, this is not an intentional change (refactored by IDE). I'll revert it.